src/ol/interaction.js

```diff
@@ -155,3 +134,26 @@ export function defaults(opt_options) {
   return interactions;
 
 }
+
+
+export {default as DoubleClickZoom} from './interaction/DoubleClickZoom.js';
+export {default as DragAndDrop} from './interaction/DragAndDrop.js';
+export {default as DragBox} from './interaction/DragBox.js';
+export {default as DragPan} from './interaction/DragPan.js';
+export {default as DragRotate} from './interaction/DragRotate.js';
+export {default as DragRotateAndZoom} from './interaction/DragRotateAndZoom.js';
+export {default as DragZoom} from './interaction/DragZoom.js';
+export {default as Draw} from './interaction/Draw.js';
+export {default as Extent} from './interaction/Extent.js';
+export {default as Interaction} from './interaction/Interaction.js';
+export {default as KeyboardPan} from './interaction/KeyboardPan.js';
+export {default as KeyboardZoom} from './interaction/KeyboardZoom.js';
+export {default as Modify} from './interaction/Modify.js';
+export {default as MouseWheelZoom} from './interaction/MouseWheelZoom.js';
+export {default as PinchRotate} from './interaction/PinchRotate.js';
+export {default as PinchZoom} from './interaction/PinchZoom.js';
+export {default as Pointer} from './interaction/Pointer.js';
+export {default as Select} from './interaction/Select.js';
+export {default as Snap} from './interaction/Snap.js';
+export {default as Translate} from './interaction/Translate.js';
+export {defaults};
```

Comment:
I'm curious if something about the documentation process requires that export statements be grouped at the bottom.  Feels like a fragile thing to have to maintain.